app/src/main/java/org/exoplatform/tool/JavaScriptInterface.java

```diff
@@ -0,0 +1,72 @@
+package org.exoplatform.tool;
```

Comment:
Can you add the java header related to exoplatform repository ? with AGPL licence
Thanks

Reply:
Done